doc/tutorials/crypto.md

```diff
@@ -0,0 +1,106 @@
+# Cryptographic Methods in Elektra
+
+Elektra can protect the following aspects of your configuration:
+
+1. confidentiality, and
+2. integrity.
+
+Elektra provides two plugins to achieve this protection:
+
+1. `crypto`, and
+2. `fcrypt`.
+
+## Configuration File Encryption/Decryption
+
+The `fcrypt` plugin enables the encryption and decryption of entire configuration files, thus protecting the confidentiality of the configuration values.
```

Comment:
suggestions:

- of all containing configuration settings.
- of the configuration keys and values.